src/constants/url.js

```diff
@@ -0,0 +1 @@
+export const BASE_URL = 'https://hacker-news.firebaseio.com/v0';
```

Comment:
Nice to have this thing in **.env** file. Because you are hardcoding API url.

You can research on `dotenv` package.